tests/runtime/engine/parser.py

```diff
@@ -105,7 +105,7 @@ def __read_test_struct(test, test_suite):
         after = ''
         kernel_min = ''
         kernel_max = ''
-        requirement = ''
+        requirement = set()
```

Comment:
Is set ordered? Should we use a list to make it more predictable?

Reply:
Good point, it may come useful to exec the conditions in the same order they were supplied. Fixed.